controllers/user-controller.js

```diff
@@ -0,0 +1,9 @@
+const User = require('../models/User')
+
+exports.getAllUsers = async (req, res, next) => {
+  const {} = req.query
+
+  const foundUsers = await User.findMany()
+
+  return res.json({ users: foundUsers })
```

Comment:
You could make use of a javascript shorthand here by renaming `foundUsers` to the same name as your key `users` allowing you to send `{ users }`

Reply:
I'm strongly against this myself. Having dealt with 10+ controllers in a script before, it really helps mentally to know exactly what the DB entities represent. I don't want all 10 to basically stipulate res.json({users}) aha